modules/core/shared/src/main/scala/gem/GuideGroup.scala

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+
+import cats._
+import cats.implicits._
+import cats.data.NonEmptyList
+import gem.enum.Guider
+
+/**
+ * Associates one or more guide probes or guide windows with guide stars.
+ */
+sealed trait GuideGroup extends Product with Serializable {
+
+  def guideStars: NonEmptyList[(Guider, Target)]
```

Comment:
Would it make sense for this to be `NonEmptyMap`? It's implemented with a `TreeMap` so we get a canonical ordering by `Guider`.